web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1582,6 +1584,10 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   private def logSnippetFailure(sf: SnippetFailure) = logger.info("Snippet Failure: " + sf)
 
+  val throwSettingsException: LiftRulesSettingException => Unit = e => throw e
+  val warnOnSettingsException: LiftRulesSettingException => Unit = e => logger.warn("LiftRules setting safety violation!!!", e)
```

Comment:
Given the simplicity of these, I feel like we should just pick the one we want as default and let the user update it as they desire.